packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -714,24 +717,23 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 		vote: true,
 	};
 
-	async findManyByRangeQuery(query: ExecutionSummaries.RangeQuery): Promise<ExecutionSummary[]> {
-		if (query?.accessibleWorkflowIds?.length === 0) {
-			throw new ApplicationError('Expected accessible workflow IDs');
-		}
-
-		const qb = this.toQueryBuilderWithAnnotations(query);
-
-		// FIXME: This needs refactoring
-		const rawExecutionsWithTags: Array<
+	/**
+	 * This function reduces duplicate rows in the raw result set of the query builder from *toQueryBuilderWithAnnotations*
+	 * by merging the tags of the same execution annotation.
+	 * @param rawExecutionsWithTags
+	 * @private
```

Comment:
We can remove these. `private` is already specified using TS and no need to list the parameters in JSDoc, unless there's a need to elaborate it